packages/react/src/tools/__tests__/keyboard-navigation-test.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * @jest-environment jsdom
+ */
+import { getNextIndex } from '../keyboard-navigation';
+
+describe('getNextIndex', () => {
+  let indexToTest;
```

Comment:
Something fun to try here may be table tests, it could look like:

```js
const tests = [
  [
    'increments the index forward',
    {
      key: 'ArrowRight',
      index: 0,
      length: 3,
    },
    1
  ],
  // ...
];

test.each(tests)('%s', (_, { key, index, length }, expected) => {
  expect(getNextIndex(key, index, length)).toBe(expected);
});
```

If not, then we might want to update the tests to be using `test` instead of `it` since we're not doing BDD in the file. In other words:

```js
  test('increments the index forward', () => {
    expect(getNextIndex('ArrowRight', 0, 3)).toEqual(1);
  });
```

Reply:
I've never heard of Table Tests, so for the sake of getting this thing buttoned up I'll go with the second option, but is a Table Test just a test using the `.each`?